tests/Feature/HasBinaryUuidTest.php

```diff
@@ -120,6 +120,20 @@ public function it_prevents_double_encoding()
         $this->assertEquals($encodeUuid, $decodedUuid);
     }
 
+    /** @test */
+    public function it_generates_valid_routes()
+    {
+        $uuid = Uuid::uuid1();
+        $model = $this->createModel($uuid);
```

Comment:
In some tests, these two lines are put together because there's a whole list of them. But in this case, I'd add a line break here.